docs/release-notes/cli-user-edit.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  CLI: A new CLI command has been added ``det user edit <target_user> [--display-name]
+   [--remote/--no-remote] [--activate/--deactivate] [--admin/--no-admin] [--username]`` that allows
+   the user to edit mutliple fields for the target user. Old methods for editing users will still be
+   available, but depricated.
```

Comment:
:orphan:

**New Features**

-  CLI: Add a new CLI command ``det user edit <target_user> [--display-name]
   [--remote/--no-remote] [--activate/--deactivate] [--admin/--no-admin] [--username]`` that allows
   the user to edit mutliple fields for the target user. Old methods for editing users will still be
   available, but deprecated.

Reply:
Fixed typo, depricated -> deprecated